examples/common/python/utility/hex_utils.py

```diff
@@ -25,8 +25,18 @@ def pretty_ids(ids):
 def hex_to_utf(binary):
```

Comment:
LGTM says there is a unused import on line 16:
```

import string
--
  **| NEW Import of 'string' is not used.**
```

(for details, see https://lgtm.com/projects/g/hyperledger-labs/trusted-compute-framework/rev/pr-42871d48af4833557e75cb15523d85dbcdd222cc )

Reply:
Removed unused import